.github/ISSUE_TEMPLATE/general_issue.yml

```diff
@@ -25,8 +25,8 @@ body:
         - label: Edge
         - label: Safari
         - label: other
-      validations:
-        required: true
+    validations:
+      required: true
   - type: dropdown
```

Comment:
There are two other spots in this file that need `validations` moved to the correct spot.

Reply:
dang, done.